app/static/src/scss/includes/_browse-all.scss

```diff
@@ -158,6 +158,14 @@
     margin-top: 2rem;
     margin-right: 0.6rem;
   }
+
+  &--browse-all-breadcrumb {
```

Comment:
looks like this is largely the same as the styling in

```
      text-decoration: none;
      font-weight: bold;
      color: black;
      -webkit-text-fill-color: black;
      font-size: 1.5rem;
      line-height: 1.3rem;
      cursor: default;
```

since thats where this text used to be.

Can we create a higher level class that can be used by both instead of duplicating?

Reply:
we discussed offline the ability to abstract the shared styles into a mixing that can be includes in both classes:

https://www.w3schools.com/sass/sass_mixin_include.php